user.go

```diff
@@ -36,3 +36,45 @@ func (s *System) GetUser() ([]types.User, error) {
 
 	return user, nil
 }
+
+// GetUserByID returns a specific user based on it's user id
+func (s *System) GetUserByID(userID string) (*types.User, error) {
```

Comment:
Is it possible to get user by name? If so, please add a method for the same or update the existing method to GetUserByIDName(id string, name string). You can refer to this, https://github.com/dell/goscaleio/blob/main/fs.go#L54

Reply:
changed the exiting method to incorporate the name field also